utils/getTrustedCertificates.ts

```diff
@@ -0,0 +1,107 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as fse from 'fs-extra';
+import * as https from 'https';
+import * as path from 'path';
+import * as vscode from 'vscode';
+import { callWithTelemetryAndErrorHandling, IActionContext } from "vscode-azureextensionui";
+import { ext } from '../extensionVariables';
+import { globAsync } from '../helpers/async';
+import { isLinux, isMac, isWindows } from '../helpers/osVersion';
+
+let _systemCertificates: (string | Buffer)[] | undefined;
+
+export async function getTrustedCertificates(): Promise<(string | Buffer)[]> {
+    // tslint:disable-next-line:no-function-expression
+    return callWithTelemetryAndErrorHandling('docker.certificates', async function (this: IActionContext): Promise<(string | Buffer)[]> {
+        this.suppressTelemetry = true;
+
+        let systemCerts = getCertificatesFromSystem();
+
+        let certificatePaths = vscode.workspace.getConfiguration('docker').get<string[] | undefined>('certificates');
+        if (certificatePaths === undefined) {
+            if (isLinux()) {
+                certificatePaths = ['/etc/ssl/certs/ca-certificates', '/etc/openssl/certs', '/etc/pki/tls/certs', '/usr/local/share/certs'];
+            } else {
+                certificatePaths = [];
+            }
+        }
+        let folderCerts = await getCertificatesFromPaths(certificatePaths);
+
+        this.properties.fromSystem = String(systemCerts.length);
```

Comment:
nit: `fromSystemCount`?

Reply:
Seems unnecessary but done.

Reply:
Mainly concerned that it looked like we were sending cert info in telemetry. Count makes it seem obvious that it's just a non-important number